engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/content-view-details.controller.js

```diff
@@ -36,8 +36,7 @@ angular.module('Bastion.content-views').controller('ContentViewDetailsController
             publish: "Actions::Katello::ContentView::Publish",
             promotion: "Actions::Katello::ContentView::Promote",
             removal: "Actions::Katello::ContentView::Remove",
-            incrementalUpdate: "Actions::Katello::ContentView::IncrementalUpdates",
-            export: "Actions::Katello::ContentViewVersion::Export"
+            incrementalUpdate: "Actions::Katello::ContentView::IncrementalUpdates"
```

Comment:
PR LGTM but I have a question about the removal here. This code relates to the History tab on the CV details page which shows details about all of the tasks listed here, so this change means it won't show exports any more.

If it's a drop-in replacement, I think it's worth it to just change to the new task name `::Actions::Pulp3::Orchestration::ContentViewVersion::Export`

Alternatively, I see a good argument for keeping the Actions::Katello::ContentViewVersion::Export task and just make it plan `::Actions::Pulp3::Orchestration::ContentViewVersion::Export` and revert the change in this file. The main benefit being not mentioning "pulp3 orchestration" as a top-level task from Katello. 

Note there was also a BZ for it, so removing it could break QE automation: https://bugzilla.redhat.com/show_bug.cgi?id=1311526

Thoughts?

Reply:
I like the 2nd approach better, since it allows us to keep the name which won't break anything for QE/users and we can just under the hood have it plan the `::Actions::Pulp3::Orchestration::ContentViewVersion::Export` Will update and request a review when I am finished.

Reply:
I propose completely renaming `::Actions::Pulp3::Orchestration::ContentViewVersion::Export` to ` Actions::Katello::ContentViewVersion::Export` because pulp3 is the only way to export anyway. It makes much more sense in the top level Katello actions directory . We don't need to worry about renaming sub actions, just the top level orchestrate.

Reply:
I'm good with that